Makefile

```diff
@@ -35,3 +35,7 @@ clean-data:
 .PHONY: describe-data
 describe-data:
 	python -m ${BASEDIR} --describe-data
+
+.PHONY: logistic-regress
+logistic-regress:
+	python -m ${BASEDIR} --logistic-regress
```

Comment:
Add a new line at the end of the file and add the `--logistic-regress` to the end of the `run` command.